dnf.spec

```diff
@@ -1,4 +1,4 @@
-%global hawkey_version 0.13.0
+%global hawkey_version 0.13.1
```

Comment:
Please increase version of dnf because I suggest that the change will make a problem for combination of a new libdnf with old dnf.

Reply:
dnf version bumped.